src/User/PostUserDeleteActionPerformerInterface.php

```diff
@@ -23,18 +23,22 @@
 use Drupal\user\UserInterface;
 
 /**
- * Contract for triggering an Apigee specific reaction on user removal.
+ * Contract for triggering Apigee specific reactions _after_ user removal.
+ *
+ * It is important to know that when implementations are called the user
+ * entity is already removed and the database transaction is closed, there
+ * is no way to roll that back.
  */
-interface UserRemovalHandlerInterface {
+interface PostUserDeleteActionPerformerInterface {
```

Comment:
Is is better? I mean, naming is hard... but I think it is better to be explicit and descriptive in names (in domain driven style) when are designing public APIs.